arch/x86/kvm/pkvm/pgtable.h

```diff
@@ -0,0 +1,105 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#ifndef __PKVM_X86_PGTABLE_H
+#define __PKVM_X86_PGTABLE_H
+
+#include <linux/types.h>
+
+struct pkvm_pgtable;
+
+/*
```

Comment:
Make these comments compatible with kerneldoc (https://docs.kernel.org/doc-guide/kernel-doc.html) i.e. use `/**` etc?

(since they are already almost compatible with it)

This applies to all these comments in pgtable.h as well as some others (`struct pkvm_init_ops` in init_finalize.h, `bool pkvm_find_addr_range()` in memory.h).